src/scripts.js

```diff
@@ -219,6 +253,21 @@ const checkClick = (event) => {
   }
 };
 
+const beginLogin = () => {
+  let nav = document.querySelector("nav");
+  let welcome = document.querySelector(".welcome-wrapper");
+  let arrowOne = document.querySelector(".one");
+  let login = document.querySelector(".login-wrapper");
+  let tripForm = document.querySelector(".add-trip-form");
+
+  removeHidden(nav);
+  removeHidden(welcome);
+  removeHidden(arrowOne);
+  removeHidden(tripsPage);
+  removeHidden(tripForm);
```

Comment:
Great use of reusable code (‘utility function’), next time think of how you might be able to refactor an array of querySelectors to only call `removeHidden`, that way you would only have to call it once. Something to consider!